_specs/ecip-1061.md

```diff
@@ -1,18 +1,22 @@
 ---
 lang: en
 ecip: 1061
-title: Aztlán EVM and Protocol Upgrades
-status: Draft
+title: Aztlán EVM and Protocol Upgrades (Yingchun Edition)
+status: Last Call
 type: Meta
-author: Talha Cross (@soc1c)
+author: Talha Cross (@soc1c), Wei Tang (@sorpaas)
 created: 2019-06-06
 discussions-to: https://github.com/ethereumclassic/ECIPs/issues/81
 ---
 
 ### Simple Summary
 
 Enable the outstanding Ethereum Foundation _Istanbul_ network protocol upgrades on the Ethereum
-Classic network in a hard-fork code-named _Aztlán_ to enable maximum compatibility across these networks.
+Classic network without any gas-cost assumptions in a hard-fork code-named _Aztlán_ to enable 
+maximum compatibility across these networks. 
+
+The _Yingchun_ Edition refers to the preferred variant of _Aztlán_ without any repricing of state-trie 
+dependent opcodes as previously proposed in ECIP-1072.
```

Comment:
I would add a github link to the ECIP-1072 in question, but other than that, this LGTM